components/MainNavBar.vue

```diff
@@ -54,7 +54,7 @@
          ] },
          { name : 'Investigations', url : '#', external: false, subpages : [
              { name: 'Case Characterisations', url: '/', external: false },
-             { name: 'Witches Meetings', url: '/', external: false },
+             { name: 'Witches Meetings', url: '/CaseInformation', external: false },
```

Comment:
It would be good if the url was case-information please

Reply:
I've also noticed that when I click on links, the URL is no longer changing. Could you look into this please? Thanks Ruby

Reply:
have changed to case-information

Reply:
is that not because currently for all the pages not added yet, they currently link to '/' (the index page)

for me the url is changing for the pages that have been added?

Reply:
Sorry Ruby, I'm not sure you saw my edit about changing to witches-meetings. This would be preferred I think just so we keep namings the same.

The only thing I'm wondering about is can we have the url as witches-meetings but the page called WitchesMeetings or does it have to match? I don't think it looks good to have kebab case for component files.

Reply:
had a quick look and does look like the file name has to match the url so happy to keep as is. Thanks Ruby